apps/front-end/src/components/panel/directoryPanel/DirectoryPanel.tsx

```diff
@@ -40,17 +39,12 @@ const DirectoryPanel = () => {
 
   const directoryOptions = useAppSelector(selectDirectoryOptions);
   const activeValue = directoryOptions.value;
-
-  const [resultsTotals, setResultsTotals] = useState<number[]>([]);
+  const [resultsTotals, setResultsTotals] = useState<any>();
 
   useEffect(() => {
-    //this is where i will manipulate my list of features
-
-    fetchResults().then((results) => {
-      console.log(results);
+    fetchResults().then((results: any) => {
+      if (!results.failed) setResultsTotals(results);
```

Comment:
My typechecker says `results` could be undefined here, in which case trying to get `results.failed` may be an exception